internal/connector/test/integration/features/connector-agent-api.feature

```diff
@@ -7,8 +7,7 @@ Feature: connector agent API
   Background:
     Given the path prefix is "/api/connector_mgmt"
     Given an org admin user named "Jimmy"
-    Given I store orgid for "Jimmy" as ${jimmy_org_id}
-    Given an org admin user named "Bobby" in organization "${jimmy_org_id}"
+    Given an org admin user named "Bobby" in organization "${Jimmy.OrgId}"
```

Comment:
This seems to be the only change in tests, is this sufficient to test all the changes?

Reply:
The changes are guards for bad data, which we don't have right now. We can add more negative tests in the future.

Reply:
See issue https://issues.redhat.com/browse/MGDCTRS-695 for future enhancements.